samples/template/scripts/create_template.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+import glob
+import json
+import os
+import pathlib
+
+
+EXAMPLE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_REPO_DIR = os.path.dirname(os.path.dirname(EXAMPLE_DIR))
+TIDES_SPEC = os.path.join(BASE_REPO_DIR, "spec")
+SCHEMAS = glob.glob(os.path.join(TIDES_SPEC, "**/*.schema.json"), recursive=True)
+SCHEMAS_LOC = "https://raw.githubusercontent.com/TIDES-transit/TIDES/main/spec/"
+
+# DATAPACKAGE.JSON INFORMATION
```

Comment:
Why define a bunch of string and code to write strings to a file? Why to just add a file? Actually, let's take this a step further, and just move the "datapackage.json" text into the readme in "/samples"? Then drop the entire template directory, with the CSVs that I don't think are useful for any purpose.